tchannel/thrift/module.py

```diff
@@ -0,0 +1,114 @@
+from __future__ import (
+    absolute_import, division, print_function, unicode_literals
+)
+
+import inspect
+import types
+
+from .reflection import get_service_methods
+
+
+def from_thrift_module(service, thrift_module):
+
+    # start building requests instance
+    maker = ThriftRequestMaker(service, thrift_module)
+
+    # create methods that mirror thrift client
+    # and each return ThriftRequest
+    methods = _create_methods(thrift_module)
+
+    # attach to maker
+    for name, method in methods.iteritems():
+        method = types.MethodType(method, maker, ThriftRequestMaker)
+        setattr(maker, name, method)
```

Comment:
This is doing two separate out-of-context loops for no reason. Can you get rid of `create_methods` but keep `create_method`.